app/src/main/java/au/com/shiftyjelly/pocketcasts/ui/MainActivity.kt

```diff
@@ -1340,6 +1345,13 @@ class MainActivity :
         }
     }
 
+    private fun openReferralClaim(code: String) {
+        // TODO decide where to store the referral code
+        Timber.i("Referral code: $code")
```

Comment:
@ashiagr I wasn't sure where to save the referral code. Maybe we can do this in another PR.